BigBlueButton-RLP/View/WebNavigationView.swift

```diff
@@ -0,0 +1,87 @@
+import UIKit
+
+protocol WebNavigationViewDelegate: AnyObject {
+    func didTapBackBtn()
+    func didTapForwardBtn()
+    func didTapRefreshBtn()
+}
+
+class WebNavigationView: UIView {
+    
+    private static let spacing: CGFloat = 32
+    weak var webNavigationDelegate: WebNavigationViewDelegate?
+    private let stackView: UIStackView = {
+        let stackView = UIStackView(frame: .zero)
+        stackView.axis = .horizontal
+        stackView.distribution = .equalSpacing
+        stackView.alignment = .center
+        stackView.spacing = WebNavigationView.spacing
+        return stackView
+    }()
+    
+    private let backButton: UIButton = {
+        let btn = UIButton(frame: .zero)
+        let image = UIImage(systemName: "arrow.left")
```

Comment:
If we're using "leading" and "trailing" for auto-layouts (instead of left and right) then we should also account for right-to-left languages. This would mean the back button is `arrow.right` for RTL languages.

Reply:
Either that or change the constraints to use left and right.